Somehow breaks Mac OS builds? Not sure how that's related to the changes.
```
[2022-07-18T01:11:15.037Z] ld: warning: object file (../StormLib/build/libstorm.a(bn_fast_s_mp_mul_high_digs.c.o)) was built for newer macOS version (12.2) than being linked (12.0)
[2022-07-18T01:11:15.037Z] python3 copycheck.py
[2022-07-18T01:11:15.037Z] rm -rf ../OTRExporter/oot.otr
[2022-07-18T01:11:15.037Z] cd ../OTRExporter && python3 extract_assets.py
[2022-07-18T01:11:15.037Z] dyld[81462]: Symbol not found: _der_decode_boolean
[2022-07-18T01:11:15.037Z]   Referenced from: /Users/jenkins/workspace/SoH_Multibranch_PR-825/ZAPDTR/ZAPD.out
[2022-07-18T01:11:15.037Z]   Expected in: /usr/lib/libSystem.B.dylib
[2022-07-18T01:11:15.037Z] Error when building the OTR file...
[2022-07-18T01:11:15.037Z] Aborting...
[2022-07-18T01:11:15.037Z] ../ZAPDTR/ZAPD.out ed -i ../soh/assets/xml/GC_NMQ_D/ -b baserom_non_mq.z64 -fl CFG/filelists -o placeholder -osf placeholder -gsf 1 -rconf CFG/Config.xml -se OTR
[2022-07-18T01:11:15.037Z] 
[2022-07-18T01:11:15.037Z] 
[2022-07-18T01:11:15.037Z] 
[2022-07-18T01:11:15.037Z] 
[2022-07-18T01:11:15.037Z] cp ../OTRExporter/oot.otr .
[2022-07-18T01:11:15.037Z] cp: ../OTRExporter/oot.otr: No such file or directory
[2022-07-18T01:11:15.037Z] make[1]: *** [mpq] Error 1
[2022-07-18T01:11:15.037Z] make: *** [setup] Error 2
```

--- a/ZAPDTR/Makefile
+++ b/ZAPDTR/Makefile
@@ -45,8 +45,8 @@ ifneq ($(DEPRECATION_ON),0)
 endif
 # CXXFLAGS += -DTEXTURE_DEBUG
 
-LDFLAGS := -lm -ldl \
-  -L../StormLib/build -L../libultraship -pthread -lultraship ZAPDUtils/ZAPDUtils.a ../StormLib/build/libstorm.a -lbz2
+LDFLAGS := -Llib/libgfxd -L../libultraship -L../StormLib/build \
+  -pthread -lgfxd -lultraship ZAPDUtils/ZAPDUtils.a -lstorm -lbz2 -lm -ldl
 
 ifeq ($(UNAME), Darwin)
   LDFLAGS += $(shell pkg-config --libs glew libpng zlib) $(shell sdl2-config --libs) -framework OpenGL -framework Foundation
@@ -137,4 +137,4 @@ ZAPDUtils:
 
 # Linking
 ZAPD.out: $(O_FILES) lib/libgfxd/libgfxd.a ExporterTest ZAPDUtils StormLib
-	$(CXX) $(CXXFLAGS) $(O_FILES) lib/libgfxd/libgfxd.a $(EXPORTERS) $(LDFLAGS) $(OUTPUT_OPTION)
+	$(CXX) $(CXXFLAGS) $(O_FILES) $(EXPORTERS) $(LDFLAGS) $(OUTPUT_OPTION)

